### PR TITLE
Issue-#193 - Add planning colors meaning

### LINKED ISSUE #193: Add planning colors meaning
Il faudrait ajouter une légende pour expliquer les couleurs 
- du calendrier de sélection de dispo utilisateur / véhicule
- du planning global des structures

=> à voir comment présenter ça de façon discrète

### PR DESCRIPTION
**Pull request by Mathieudewet**
Closes #193

### CODE REVIEW

File: assets/css/legend.scss [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+    a {
+      color: #777;
+    }
+
+    .mobilise {
+      background: #67a2d3;
+    }
+
+    .available {
+      background: #76bd79;
```
Comment (mRoca) [inline assets/css/legend.scss:53]: you should use the same class as for boxes. Or at least some sass variable

File: assets/css/legend.scss [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+      background: #ff767f;
+    }
+
+    .unavailable-for-mobilization {
+      background-color: #7a7a7a !important;
+    }
+  }
+}
+
+@media screen and (min-width: 451px) {
```
Comment (mRoca) [inline assets/css/legend.scss:66]: you can remove this media query, as it's the default one

File: assets/css/legend.scss [hunk 3]
```diff
@@ -0,0 +1,118 @@
+@media screen and (max-width: 450px) {
```
Comment (mRoca) [inline assets/css/legend.scss:1]: you can define all media query rules inside the default ones : http://thesassway.com/intermediate/responsive-web-design-in-sass-using-media-queries-in-sass-32

File: assets/css/legend.scss [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+    background: #76bd79;
+  }
+
+  .unavailable {
+    background: #ff767f;
+  }
+
+  .unavailable-for-mobilization {
+    background-color: #7a7a7a !important;
+  }
```
Comment (fmeynard) [inline assets/css/legend.scss:47]: You should rebase on master and use/modify selector for centralized classes from [availability-table.scss](https://github.com/crf-devs/resop/blob/master/assets/css/availability-table.scss)

```
.planning-actions-container,
table.availability-table {
  .unknown {
    background-color: #ff767f;
    color: white;
  }

  .available {
    background-color: #76bd79;
    color: white;
  }

  .booked {
    background-color: #67a2d3;
    color: white;
  }

  .locked {
    background-color: #7a7a7a;
    color: white;
  }
}
```

File: templates/availability/_table.html.twig [hunk 5]
```diff
@@ -60,6 +60,19 @@
     </table>
 </div>
 
+<div class="row">
+    <div class='legend'>
+        <div class='legend-scale'>
+            <ul class='legend-labels'>
```
Comment (fmeynard) [inline templates/availability/_table.html.twig:66]: Doubles quotes

**Comment by mRoca** [comment]
I think we should define all those colors as variables in the app.scss file : https://sass-lang.com/documentation/variables

**Comment by mRoca** [comment]
I'm closing this PR in favor of #263

**Pull request closed without merging**